extra-resources/mods/Cyborg-Chronicles/Game.groovy

```diff
@@ -258,17 +263,6 @@ rules {
             // ...return from this method without doing anything
             return
         }
-        // CYBORG-CHRONICLES MECHANIC:
```

Comment:
I don't remember why I deleted that, come to think of it. I'll look for a commit message. Could you try adding it back in and try to run the tests?